server/backend/migration/20231003000000.go

```diff
@@ -0,0 +1,193 @@
+package migration
+
+import (
+	embed "embed"
+	"encoding/json"
+	"gorm.io/gorm/logger"
+
+	"github.com/TUM-Dev/Campus-Backend/server/backend"
+	"github.com/TUM-Dev/Campus-Backend/server/model"
+	"github.com/go-gormigrate/gormigrate/v2"
+	"github.com/guregu/null"
+	log "github.com/sirupsen/logrus"
+	"gorm.io/gorm"
+)
+
+type multiLanguageTags struct {
+	MultiLanguageTags []tag `json:"tags"`
+}
+type tag struct {
+	TagNameEnglish string `json:"tagNameEnglish"`
+	TagNameGerman  string `json:"tagNameGerman"`
+}
+
+type multiLanguageNameTags struct {
+	MultiLanguageNameTags []nameTag `json:"tags"`
+}
+type nameTag struct {
+	TagNameEnglish string   `json:"tagNameEnglish"`
+	TagNameGerman  string   `json:"tagNameGerman"`
+	NotIncluded    []string `json:"notincluded"`
+	CanBeIncluded  []string `json:"canbeincluded"`
+}
+
+//go:embed static_data
+var staticData embed.FS
+
+// migrate20231003000000
+// migrates the static data for the canteen rating system and adds the necessary cronjob entries
+func (m TumDBMigrator) migrate20231003000000() *gormigrate.Migration {
+	return &gormigrate.Migration{
+		ID: "20231003000000",
+		Migrate: func(tx *gorm.DB) error {
+			if err := tx.Delete(&model.DishNameTagOption{}, &model.CafeteriaRatingTagOption{}, &model.DishNameTagOptionIncluded{}, &model.DishNameTagOptionExcluded{}).Error; err != nil {
```

Comment:
note that because this migration is referencing models, this is not a stable migration => this migration might break in the future, depending on changes to the models.
Please consider adding these models to this migration (code duplication is good in this case imo)